hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -3274,18 +3274,18 @@ private void setupResponse(RpcCall call,
     cos.writeRawByte((byte)((length >>> 16) & 0xFF));
     cos.writeRawByte((byte)((length >>>  8) & 0xFF));
     cos.writeRawByte((byte)((length >>>  0) & 0xFF));
-    cos.writeRawVarint32(header.getSerializedSize());
+    cos.writeUInt32NoTag(header.getSerializedSize());
     header.writeTo(cos);
     if (payload != null) {
-      cos.writeRawVarint32(payload.getSerializedSize());
+      cos.writeUInt32NoTag(payload.getSerializedSize());
```

Comment:
Ditto

Reply:
Ditto :) All variations of *\*RawVarint\*()* APIs are deprecated and replaced with *\*UInt\*NoTag()* APIs.
Deprecated APIs internally call *\*UInt\*NoTag()* API itself.